src/config/commands/susList.ts

```diff
@@ -32,4 +34,6 @@ export const SusColours = [
   "#928776",
   "#738491",
   "#efc3cf",
+  "#ff7f50",
+  "#ffe135", 
```

Comment:
```suggestion
  "#d76464",
  "#fffebe", 
```

These are the hex codes I grabbed from the image.

Reply:
Sure @nhcarrigan I will use the following hex codes and will make an PR which will include the update of the import statement. For now, you can close this PR, gonna make another one.

Reply:
You do not need to make a new PR, simply push additional commits to your `patch-1` branch and this PR will automatically update.

Reply:
@nhcarrigan Okay yeah sure.

Reply:
Hey @nhcarrigan I have made the additional changes to the `patch-1` branch. Do I also need to modify the Pull request template?